docs/ops/condition/If_8.md

```diff
@@ -0,0 +1,226 @@
+## If <a name="If"></a> {#openvino_docs_ops_infrastructure_If_8}
+
+**Versioned name**: *If-8*
+
+**Category**: Infrastructure
+
+**Short description**: *If* operation contains two internal networks(subgraphs) such as `then_body` and `else_body`, 
+and performs one of them depending on `cond` value. If `cond` is  `True`, `then_body` is executed. If `cond` is  `False`, 
+the operation executes the `else_body` subgraph. 
+
+**Detailed description**
+
+*If* must not contain empty subgraphs. Each of them must have at least one operation `Result`. 
+Also the number of outputs from *If* always must be greater than zero and equal to the number of outputs from each subgraph.
+
+**If attributes**:
+
+* **Subgraphs**:
+
+    `then_body`/`else_body` are subgraphs that are executed depending on the `cond` value. 
+    The subgraph is described operation by operation as a typical IR network. 
+    The subgraph has parameters (`Parameter` operations) and results (`Result` operations).
+    
+    * **Subgraph's parameters** - inputs to the subgraph which associated with *If* inputs via *port_map*. 
+    The subgraph can have any number of parameters (even zero).
```

Comment:
Can we use here word "input" instead of "parameter"? I think that the word "input" explains what is this node for much better than "parameter".

Reply:
done